tests/UKHO.ERPFacade.WebJob.UnitTests/Services/MonitoringServiceTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using FakeItEasy;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using UKHO.ERPFacade.Common.IO;
+using UKHO.ERPFacade.WebJob.Services;
+
+namespace UKHO.ERPFacade.WebJob.UnitTests.Services
+{
+    [TestFixture]
+    public class MonitoringServiceTests
+    {
+        private ILogger<MonitoringService> _fakeLogger;
+        private IAzureTableReaderWriter _fakeAzureTableReaderWriter;
+
+        private MonitoringService _fakeMonitoringService;
+
+        [SetUp]
+        public void Setup()
+        {
+            _fakeLogger = A.Fake<ILogger<MonitoringService>>();
+            _fakeAzureTableReaderWriter = A.Fake<IAzureTableReaderWriter>();
+            _fakeMonitoringService = new MonitoringService(_fakeLogger, _fakeAzureTableReaderWriter);
+        }
+
+        [Test]
+        public void WhenWebJobStartsAndMonitorsIncompleteTransactions_ThenValidateEntityMethodMustbeCalled()
+        {
+            _fakeMonitoringService.MonitorIncompleteTransactions();
+
+            A.CallTo(() => _fakeAzureTableReaderWriter.ValidateEntity()).MustHaveHappened();
+        }
+
+        [Test]
+        public void WhenValidateEntityThrowsException_ThenThrowsException()
+        {
+
+            A.CallTo(() => _fakeAzureTableReaderWriter.ValidateEntity()).Throws(new NotSupportedException("Fake Exception"));
```

Comment:
Can we split tests for AzureTableReaderWriter out into a separate class, so that they are simple to find? We should make it clear what the system under test is by grouping them in this way